migen/build/xilinx/ise.py

```diff
@@ -44,7 +44,7 @@ def _build_ucf(named_sc, named_pc):
     return r
 
 
-def _build_xst_files(device, sources, vincpaths, build_name, xst_opt):
+def _build_xst_files(device, sources, build_name, xst_opt):
     prj_contents = ""
     for filename, language, library in sources:
         prj_contents += language + " " + library + " " + tools.cygpath(filename) + "\n"
```

Comment:
Is ``cygpath`` still required if we're copying all inputs?

Reply:
If everything is relative, then no `cygpath` is not required.

Reply:
`cygpath` removed.